Formula/b/bashate.rb

```diff
@@ -16,16 +18,15 @@ class Bashate < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d760da6d91318d5f9e1ee0a0bdb716145bf0c9bf50531350796df64a583d1721"
   end
 
-  depends_on "python-setuptools" => :build
-  depends_on "python-pbr"
```

Comment:
if we are going to remove pbr, should we bump the revision?

Reply:
Maybe. I was revision bumping based on user input that `brew` doesn't handle removal cleanly but it may be worth testing it further to see what is exact behavior.

If `brew` is auto-removing the removed formula, then it may be worth checking if `brew` should retain installation if removed formula is listed in tab of another installed formula.

Reply:
I don't think we should rev bump until we are actually going to open the removal PR. Version bumps can happen in the meantime that would remove the need for some rev bumps.